src/ocamlutil/pretty.ml

```diff
@@ -456,32 +465,32 @@ let rec scan (abscol: int) (d: doc) : int =
 
 (** Keep a running counter of the newlines we are taking. You can read and
     reset this from user code, if you want *)
-let countNewLines = ref 0
+let countNewLines = refDLS 0
 
 (* The actual function that takes a document and prints it *)
 let emitDoc
     (emitString: string -> int -> unit) (* emit a number of copies of a
                                            string *)
     (d: doc) =
-  let aligns: int list ref = ref [0] in (* A stack of alignment columns *)
+  let aligns: int list refDLS = refDLS [0] in (* A stack of alignment columns *)
```

Comment:
I'm confused about this one. This is a local reference, but there's also a global one with the same name. How are these related?
By defining `aligns` locally, this function has no access to the global one, so it seems like this could be just a `ref` still. Or am I missing something?